### DISCUSSION TITLE
CAD formats

### DISCUSSION THREAD

**Discussion by EricHarmon** [dlemstra/Magick.NET#1170] [General] [answered]
I would love to see DXF, DWG, DWF added to the list of supported file types. Just to be able to load a DXF file and convert it to, say, a raster image format such as PNG, TIF, etc. would be tremendous.

**Comment by dlemstra** [comment, marked as answer]
It looks like the `uniconvertor` executable is used for that. You can probably add this to de `delegates.xml` file yourself.

**Reply by EricHarmon** [reply]
OK, I just looked up uniconvertor. It looks like it's a standalone app rather than a library where I can simply make a method call, but it might be good enough nonetheless. I'm not sure what delegates.xml is (I'm writing a .NET 6 WPF Desktop application), but I can certainly run an external process from WPF. Thank you for the suggestion.
